README.md

```diff
@@ -124,6 +133,7 @@ The host name and port must be a unique identifier for your
 #### `options.port`
 
 The port for which `TChannel` will open a TCP server on.
+If the port is 0, the operating system will select an available port for you.
```

Comment:
there is no `options.port` or `options.host` we have to change the documentation for the `TChannel` constructor and add documentation for `listen()` method.